printer_mods/Takuya/USBCameraMount/ReadMe.md

```diff
@@ -0,0 +1,21 @@
+This my my contribution to the awesome Voron Community!
+
+Its a camera mount for a USB Camera.
```

Comment:
Missing apostrophe in `It's`.

Please specify which USB camera this is for, I don't think they all have the same mounting holes or patterns.

Reply:
Thanks,
done